services/hidconfigurator.md

```diff
@@ -0,0 +1,53 @@
+# HID Configurator
+
+    identifier: 0x1e5758b5
+    
+A service for configuring how Jacdac device map to HID input events. Users can have multiple configurations and swap between them using the `swapTo` or `next` commands.
+
+## Registers
+
+    rw num_configurations: u16 @ 0x80
+ 
+The number of configurations stored on this device.
+
+## Commands
+
+    command retrieve_configuration @ 0x80 {
+        configuration_number: u16
```

Comment:
is number supposed to be unique identifier?

Reply:
Nope, just an index (i.e. 0 is the first one, 1 is the second one, etc.)

I'll add a comment to remove ambiguity

Reply:
is u16 for alignment purposes?

Reply:
Not necessarily... u16 is a bit much... u8 would be fine!